File: Leetcode/268. Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int ans = 0,i;
        for(i=0;i<nums.size();i++)
        ans = ans ^ nums[i] ^ i;
        ans ^= nums.size();
        return ans;
    }
};

File: Leetcode/268. Missing Number.java
https://leetcode.com/problems/missing-number/submissions/
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int ans = n;
        for(int i=0;i<n;i++){
            ans ^= nums[i]^i;
        }
        return ans;
    }
}
